src/views/Layout.jsx

```diff
@@ -11,11 +15,16 @@ import './Layout.css';
  */
 
 export function Layout() {
+	const { isOpen, onOpen, onClose } = useDisclosure();
 	return (
 		<>
+			<About isOpen={isOpen} onClose={onClose} />
 			<div className="Layout">
 				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
+					<Heading as="button" onClick={onOpen} cursor={'pointer'}>
+						Smart shopping list
+						<VisuallyHidden>Open about modal</VisuallyHidden>
+					</Heading>
```

Comment:
Unless I read our PR description, I was initially confused where we could find the about page. We spoke during OH about using a button above the add items button. Are we able to use that, just because its clearer?

Reply:
It was looking cluttered at the top of the app with both of the buttons (adding an item and also learning about the app) so we went with a subtle easter egg instead. We added the a cursor when you hover over the title and talked about also changing the title color on hover as clues that it is clickable.